extensions/todolist@bsaleil.org/Makefile.am

```diff
@@ -0,0 +1,14 @@
+todoschemadir = $(extensiondir)/schemas
```

Comment:
@krnowak you mean instead of generating the `gschemas.compiled` we just replace the code that expects the already compiled one? Will give it a try, but not sure what that will break. :D